OmiseSDKTests/Compatability Suite/OmiseTokenRequestTest.swift

```diff
@@ -4,7 +4,7 @@ import OmiseSDK
 
 @available(*, deprecated)
 class OmiseTokenRequestTest: XCTestCase {
-    private let publicKey = "<#Omise Public Key#>"
+    private let publicKey = "pkey_test_58wfnlwoxz1tbkdd993"
```

Comment:
Should this somehow be stored in an environment variable? 🤔

Reply:
Yes I think that would be nice. Now we have this key in three different files. 
Is it possible to securely store keys in GitHub Actions so we can remove it from the repository?

Reply:
Yes 

https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets

Reply:
Okay nice. But I think there is not straightforward way to set the environment variables from xcodebuild. Let's look into this later and create another PR.

Reply:
Seems to look similar to the environment variable in Node.js 😄